cmake/CMakeLists.txt

```diff
@@ -1801,6 +1801,7 @@ if (onnxruntime_USE_CUDA)
       set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_53,code=sm_53") # TX1, Nano
       set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_62,code=sm_62") # TX2
       set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_72,code=sm_72") # AGX Xavier, NX Xavier
+      set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode=arch=compute_87,code=sm_87") # AGX Orin, NX Orin
```

Comment:
Just curious - does this mean the binary size of the onnxruntime .dll and .so will now increase as there is now support for one more arch and possibly more PTX code (due to the `gencode` flag) ?

Reply:
No. These are controlled by Azure Devops Yaml files.

Reply:
Ok, those are for the release binaries, but for anyone building from source (with no options provided) ?

Reply:
yes, that's right. someone building from source on a jetson device doesn't need to add extra args.